src/modules/ROOT/pages/changelog/202309/05.json

```diff
@@ -0,0 +1,110 @@
+{
+  "meta": {
+    "releasedOn": "2024.03.21"
+  },
+
+  "added": [
+    {
+      "what": [
+        "the option to access user information like username, handle, email, etc. in pages, charts, and record fields."
```

Comment:
Either provide the entire list of options or link to somewhere where all of the options are listed (like [a type reference](https://docs.cortezaproject.org/corteza-docs/2023.9/integrator-guide/expr/type-reference.html#objref-user))